assignment3/dictionary.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/python3.5
+
+class Dictionary(object):
+    """A dictionary class holding a word dictionary as a list
+    and implements the two functions isWord and isPrefix. """
+
+    def __init__(self, wordList):
+        """The dictionary contains words given in the wordList. 
+        It also constructs a prefixSet given the wordList.
+        No duplicates exist in wordSet and prefixSet. 
```

Comment:
Thanks for documenting the invariants the class ensures. The better defined the behavior the easier it is to understand the code, reason about behavior and later maintain it :)